.travis.yml

```diff
@@ -4,8 +4,14 @@ node_js:
   - 6
   - 5
   - 4
+  - 0.12.0
+branches:
```

Comment:
Travis will only build from master. This will allow us to create a dev branch that can be updated continuously.

Every tiny change on NPM must be reflected by a change in its version number. This way _you_ are the only one in charge of when and how the module gets updated.